src/sentry/analytics/events/issue_resolved.py

```diff
@@ -6,6 +6,7 @@ class IssueResolvedEvent(analytics.Event):
 
     attributes = (
         analytics.Attribute("user_id", required=False),
+        analytics.Attribute("project_id", required=False),
```

Comment:
Made `project_id` not required in case there was a reason it was not included in the first place